functests/4_latency/latency.go

```diff
@@ -235,45 +253,42 @@ var _ = Describe("[performance] Latency Test", func() {
 })
 
 //read LATENCY_TEST_RUN environment var and parse it if set
-func getLatencyTestRun() (parsedValue bool) {
+func getLatencyTestRun() (parsedValue bool, errMsg error) {
+	var err error
 	val := false
-	latencyTestRunEnv := os.Getenv("LATENCY_TEST_RUN")
-	if latencyTestRunEnv != "" {
-		if latencyTestRunEnv == "true" {
-			val = true
-		}
+	if latencyTestRunEnv, ok := os.LookupEnv("LATENCY_TEST_RUN"); ok {
+		val, err = strconv.ParseBool(latencyTestRunEnv)
 	}
-	return val
+	return val, err
 }
 
 //read LATENCY_TEST_RUNTIME environment var and parse it if set
 func getLatencyTestRuntime() (parsedValue string) {
-	latencyTestRuntimeEnv := os.Getenv("LATENCY_TEST_RUNTIME")
-	return latencyTestRuntimeEnv
+	if latencyTestRuntimeEnv, ok := os.LookupEnv("LATENCY_TEST_RUNTIME"); ok {
+		return latencyTestRuntimeEnv
+	}
+	return defaultTestRuntime
 }
 
 //read LATENCY_TEST_DELAY environment var and parse it if set
 func getLatencyTestDelay() (parsedValue int, errMsg error) {
 	var err error
-	var val int
-	latencyTestDelayEnv := os.Getenv("LATENCY_TEST_DELAY")
-	if latencyTestDelayEnv != "" {
+	val := defaultTestDelay
+	if latencyTestDelayEnv, ok := os.LookupEnv("LATENCY_TEST_DELAY"); ok {
 		if val, err = strconv.Atoi(latencyTestDelayEnv); err != nil {
-			err = fmt.Errorf("the environment variable LATENCY_TEST_DELAY has incorrect value %q", latencyTestDelayEnv)
+			err = fmt.Errorf("the environment variable LATENCY_TEST_DELAY has incorrect value %q: %w", latencyTestDelayEnv, err)
```

Comment:
this is good. Even better: make it look like `getLatencyTestRuntime`. The skeleton code is the same, it should be doable.

Reply:
@fromanirh  Should we preserve the modified error message below ?
fmt.Errorf("the environment variable LATENCY_TEST_DELAY has incorrect value %q: %w", latencyTestDelayEnv, err)